Shared.Tests/CloudinaryTest.cs

```diff
@@ -2247,6 +2309,84 @@ public void TestCreateTransform()
             Assert.AreEqual(m_simpleTransformation.Generate(), new Transformation(getResult.Info[0]).Generate());
         }
 
+        [Test]
+        public void TestCreateTransformWithExt()
+        {
+            // should allow creating a transformation containing an extension suffix (e.g., w_150,h_150/jpg)
+
+            CreateTransformParams create = new CreateTransformParams()
+            {
+                Name = m_updateTransformationWithExtAsString,
+                Transform = m_updateTransformation,
+                Format = m_extension
+            };
+
+            m_cloudinary.CreateTransform(create);
+
+            GetTransformParams get = new GetTransformParams()
+            {
+                Transformation = create.Transform.Generate(),
+                Format = m_extension
+            };
+
+            GetTransformResult getResult = m_cloudinary.GetTransform(get);
+
+            Assert.IsNotNull(getResult);
+            Assert.AreEqual(true, getResult.Strict);
+            Assert.AreEqual(false, getResult.Used);
+            Assert.AreEqual(1, getResult.Info.Length);
+
+            object format;
```

Comment:
Can be declared inline:
if (getResult.Info[0].TryGetValue("format", out var format))